opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/client/RestTemplateSpanDecorator.java

```diff
@@ -60,11 +61,21 @@
     class StandardTags implements RestTemplateSpanDecorator {
         private static final Log log = LogFactory.getLog(StandardTags.class);
 
-        public static String COMPONENT_NAME = "java-spring-rest-template";
+        public static final String COMPONENT_NAME = "java-spring-rest-template";
+
+        private final String componentName;
+
+        public StandardTags() {
+            this(null);
```

Comment:
Passing `COMPONENT_NAME` makes it cleaner, at least for me

Reply:
And I was looking at this line for at least a minute :)